.github/workflows/LinuxRelease.yml

```
@@ -211,6 +212,7 @@ jobs:
 
  linux-extensions-64-aarch64:
     name: Linux Extensions (AARCH64)
+    if: startsWith(github.ref, 'refs/tags/v') || github.ref == 'refs/heads/master'
```

```suggestion
    if: startsWith(github.ref, 'refs/tags/v') || github.ref == 'refs/heads/master' || github.repo != 'duckdb/duckdb'
```
(syntax is very much pseudo-code)
I think we basically want to restrict this (and a few other similar places) only for duckdb/duckdb, since it would be useful to have this run also from any branch on forks.